apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,331 @@
+| ---------------------- | ---------------------------------------------------- |
+| **pollDuration**       | Defines how long is going to be the poll in seconds. |
+| **coordinatorPubkey**  | Defines the coordinator public MACI key.             |
+| **useQuadraticVoting** | Defines if the poll uses quadratic voting or not.    |
+
+### Deploy MACI Contracts
+
+To deploy the MACI contracts to a specific network you can append `:network` to the deployment commands, e.g. `pnpm deploy:sepolia` - please refer to the available networks on the package.json scripts section.
+
+```bash
+pnpm deploy:NETWORK
+```
+
+There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+
+```bash
+cp default-deployed-contracts.json deployed-contracts.json
+```
+
+Delete any contract you want to redeploy and then run the following command to save gas:
+
+```bash
+pnpm deploy:NETWORK --incremental
+```
+
+## Deploy Poll
+
+To deploy your first Poll you can run the following command:
+
+```sh
+pnpm deploy-poll:NETWORK
+```
+
+:::important
+Making another poll doesn't require deploying MACI contracts again, you can deploy another poll running the `pnpm deploy-poll:NETWORK` command and then use the updated poll-id.
+:::
+
+## Poll Finalization
+
+As a coordinator, first you need to merge signups and messages (votes). Messages are stored in a queue so when the poll is over, the coordinator needs to create the tree from the queue ([AccQueue](/docs/core-concepts/merkle-trees#accumulator-queue)). This optimization is needed to reduce gas cost for voters. Then coordinator generates proofs for the message processing, and tally calculations. This allows to publish the poll results on-chain and then everyone can verify the results when the poll is over. You run a merge with:
+
+```bash
+pnpm merge:[network] --poll [poll-id]
+```
+
+Then you need to generate the proofs for the message processing, and tally calculations, we do it with the following command:
+
+```bash
+pnpm run prove:[network] --poll [poll-id] \
+    --coordinator-private-key [coordinator-maci-private-key] \
+    --tally-file ../proofs/tally.json \
+    --output-dir ../proofs/proofs/ \
+    --start-block [block-number]  \
+    --blocks-per-batch [number-of-blocks]
+```
+
+:::important
+You can improve the time of the proving by including more blocks per batch, by default you can start with 500.
+:::
+
+#### Submit On-chain
+
+Now it's time to submit the poll results on-chai and then everyone can verify the results:
+
+```bash
+pnpm submitOnChain:[network] --poll [poll-id] \
+    --output-dir proofs/ \
+    --tally-file proofs/tally.json
+```
+
+#### Tally
+
+After proofs are generated, and results tallied, the results (Tally) is written to a file. This file contains the result of a Poll. Let's take a look at one:
```

Comment:
are written

Reply:
This file contains the result of a Poll. -> remove, you just said it

Reply:
how about: "Once the proofs are generated and the results are tallied, some files store the proofs, while another file stores the results. This file, referred to as the tally file, serves as the record of the Poll's final outcome and is structured as shown in the following example:" (by chatGPT)